components/generic/SmartLink.vue

```diff
@@ -45,6 +54,14 @@
           });
         }
         return this.destination;
+      },
+      isExternalLink() {
```

Comment:
Could I propose some changes:
```js
      isExternalLink() {
        if (this.useRouterLink) return false;

        if (!process.browser) return;

        const destDomain = parseDomain(this.destination);
        const dest = [destDomain.domain, destDomain.tld].join('.');
        const currentDomain = parseDomain(window.location.href);
        const current = (currentDomain === null ? window.location.hostname : [currentDomain.domain, currentDomain.tld].join('.'));

        return dest !== current;
      }
```
... to:
1. first see if it's a router link as they are always internal
2. join domain and TLD with `.` before comparison as that's how they are in FQDNs
3. fallback to the full hostname from window.location if parse domain could not identify it, which makes it work with localhost

Reply:
I have made the changes here. Much much cleaner thank you. I left out `if (this.useRouterLink) return false;` as this logic is already being applied in the HTML and this computed property will only get called when `useRouterLink` is false.